### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by mika**
s/acccess/access/
s/agument/argument/
s/avaible/available/
s/Clearify/Clarify/
s/defintions/definitions/
s/divison/division/
s/funciton/function/
s/initalized/initialized/
s/occurrance/occurrence/
s/performace/performance/
s/postive/positive/
s/redefintions/redefinitions/
s/runing/running/
s/sofware/software/
s/splitted/split/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -349,7 +349,7 @@ and this project adheres to
   - [clang_parser] pass BPFtrace as arg instead of StructMap (a0af75f) by Matheus Marchini &lt;mat@mmarchini.me&gt;
   - [ast] introduce Identifier type to AST (389d55f) by Matheus Marchini &lt;mat@mmarchini.me&gt;
   - use CMAKE_SYSTEM_PROCESSOR when selecting whether to include x86_64 or aarch64 sources (0ea7a63) by Michał Gregorczyk &lt;michalgr@fb.com&gt;
-  - Clearify error message for mismatched llvm. (9b77fee) by George Slavin &lt;george.r.slavin@gmail.com&gt;
```
Comment (fbs) [inline CHANGELOG.md]: Currently our "changelog" is just a bunch of grouped commit subject lines so I think you should leave it as is. These are typos in the commits.

File: resources/stdint.h [hunk 2]
```diff
@@ -88,7 +88,7 @@
  *
  * To accommodate targets that are missing types that are exactly 8, 16, 32, or
  * 64 bits wide, this implementation takes an approach of cascading
- * redefintions, redefining __int_leastN_t to successively smaller exact-width
+ * redefinitions, redefining __int_leastN_t to successively smaller exact-width
```
Comment (ajor) [inline resources/stdint.h]: :D one that's not our fault! (this file was copied from Clang, typo fixed upstream too now)

File: CHANGELOG.md (hunk 1 above)
Comment (mika) [reply CHANGELOG.md]: Makes sense (was unsure about that one), fixed in new version

**Comment by fbs** [comment]
Thanks!

**Pull request merged**